src/signer/api/src/types/schnorr.rs

```diff
@@ -0,0 +1,55 @@
+//! Types for the Schnorr signing API.
```

Comment:
Note: The request and response types are the same as for the canister API, so we only need to define errors.